PLEP-0007.rst

```diff
@@ -0,0 +1,364 @@
+===============================================
+PLEP-0007 – Structure of Top-Level Sub-Packages
+===============================================
+
+.. _`10.5281/zenodo.3774573`: https://doi.org/10.5281/zenodo.3774573
+
++-------------------+---------------------------------------------+
+| PLEP              | 7                                           |
++===================+=============================================+
+| author(s)         | | Erik T. Everson                           |
+|                   | | Tulasi Parashar                           |
+|                   | | Stephen Vincena                           |
+|                   | | Nicholas A. Murphy                        |
++-------------------+---------------------------------------------+
+| contact email     | eeverson@ucla.edu                           |
++-------------------+---------------------------------------------+
+| date created      | 2019-10-27                                  |
++-------------------+---------------------------------------------+
+| date last revised | 2020-05-12                                  |
++-------------------+---------------------------------------------+
+| type              | Standard                                    |
++-------------------+---------------------------------------------+
+| status            | Accepted                                    |
++-------------------+---------------------------------------------+
+| DOI               | `10.5281/zenodo.3774573`_                   |
+|                   |                                             |
++-------------------+---------------------------------------------+
+
+**Contents**
+
+* `Abstract`_
+* `Detailed Description`_
+
+  * `Why?`_
+  * `Top-Level Sub-Packages`_
+
+* `Extensible Packages are Exempt`_
+* `Implementation`_
+* `Issues, Pull Requests, and Branches`_
+* `Backward Compatibility`_
+* `Decision Rationale`_
```

Comment:
As an aside, maybe we should add a table of contents to the PLEP template.  That way, future PLEP writers would have a draft table of contents ready.